bfe_modules/mod_tcp_keepalive/data_load.go

```diff
@@ -0,0 +1,212 @@
+// Copyright (c) 2021 The BFE Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package mod_tcp_keepalive
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net"
+	"os"
+)
+
+/*
+{
+  "Version": "x",
+  "Config": {
+    "Product1": [{
+      "VipConf": ["1.1.1.1", "1.1.1.2"],
```

Comment:
Is possible to use condition primitive  req_vip_in()/req_vip_range(), other than a field "VipConf"? 
Usually, extension module uses "cond" field to set some kind of condition for filtering, which can be set to bfe primitive expression.

Reply:
Yes bfe primitive expression satisfy this situation, I didn't use it for the following reasons:

- bfe primitive expression can describe many conditions, but in this situation only req_vip_in()/req_vip_range() are suitable, most of the primitive expressions are not ok
- there are other modules such as mod_trust_clientip like this situation(only handle vip) that not use bfe primitive expression

If using bfe primitive expression is better, I will edit the code, I have no preference.

Reply:
Thx @dblate . I'd like to suggest you may go ahead with primitive expression supported, which will provide similar interface with most of extension module. Also, it would simplify code logic in matching VIP（by cond.Match()）and retains the possibility of supporting more primitive conditions.